Makefile

```diff
@@ -19,7 +19,6 @@ coverage:
 
 release: clean
 	mkdir -p release/lib
-	cd src && $(MAKE) PREFIX=$(shell pwd)/release FORCE_RPATH='@ORIGIN/../lib' install
-	rm -f release/bin/test_log_proxy
+	cd src && $(MAKE) DESTDIR=$(shell pwd)/release/bin FORCE_RPATH='@ORIGIN/../lib' install
```

Comment:
```suggestion
	cd src && $(MAKE) PREFIX=$(shell pwd)/release FORCE_RPATH='@ORIGIN/../lib' install
```

Reply:
I propose to revert this because IMHO because it's not really compliant with: https://www.gnu.org/prep/standards/html_node/DESTDIR.html